tests/test_ocr_runner_unit.py

```
@@ -0,0 +1,72 @@
+import os
+import shutil
+import subprocess
+from pathlib import Path
+
+import pytest
+
+from scripts import ocr_runner
+
+
+class DummyProc:
+    def __init__(self, stdout=b'', returncode=0):
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)